lib/msal-browser/test/config/Configuration.spec.ts

```diff
@@ -171,6 +174,8 @@ describe("Configuration.ts Class Unit Tests", () => {
         expect(newConfig.cache.cacheLocation).to.be.eq("localStorage");
         expect(newConfig.cache.storeAuthStateInCookie).to.be.not.null;
         expect(newConfig.cache.storeAuthStateInCookie).to.be.true;
+        expect(newConfig.cache.secureCookies).to.be.not.null;
+        expect(newConfig.cache.secureCookies).to.be.true;
```

Comment:
Just need the second one, first one is not necessary.

Reply:
Although I see all the other assertions make those same two checks. Not a big deal, ultimately.

Reply:
Removing this assertion and the equivalent in the previous test. I agree they are redundant.